addons/account/models/ir_actions_report.py

```diff
@@ -54,5 +56,13 @@ def _retrieve_stream_from_attachment(self, attachment):
         vendor_bill_export = self.env.ref('account.action_account_original_vendor_bill')
         if self == vendor_bill_export and attachment.mimetype == 'application/pdf':
             record = self.env[attachment.res_model].browse(attachment.res_id)
-            return pdf.add_banner(stream, record.name, logo=True)
+            try:
+                return pdf.add_banner(stream, record.name, logo=True)
+            except ValueError:
+                record._message_log(body=_(
+                            "Error when reading the original PDF for: %r.\nPlease make sure the file is valid.",
+                            textwrap.shorten(record.name, width=100)
+                        ))
+            except PdfStreamError:
+                record._message_log(body=_("Error when adding banner to the original PDF"))
```

Comment:
```suggestion
            except ValueError or PdfStreamError:
                record._message_log(body=_(
                    "There was an error when trying to add the banner to the original PDF.\n"
                    "Please make sure the source file is valid."
                ))
```
Why not group the exceptions together ?
Also, please be careful with the indentation :wink:

Reply:
@epictete thank you for the feedback :pray: ! I updated the PR